<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>467</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelTaprendizaje">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>TAZA DE APRENDIZAJE </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEpermisible">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ERROR PERMISIBLE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTAprendizaje">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>301</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEPermisible">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>281</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEjecutar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>631</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Datos 191182</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMensaje">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>621</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelNota">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>631</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Nota: Los campos presentados solo permiten valores de tipo flotante de entre 0 y 1,&lt;/p&gt;&lt;p&gt;Ejemplo: 0.0001, 0.5, .1 etc.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEjecutar2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>631</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Datos 203411</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
